sentry-ruby/lib/sentry/log_event.rb

```diff
@@ -109,5 +126,34 @@ def value_type(value)
         "string"
       end
     end
+
+    def parameters
+      @parameters ||= begin
+        return DEFAULT_PARAMETERS unless template
+
+        parameters = template_tokens.empty? ?
+          attributes.fetch(:parameters, DEFAULT_PARAMETERS) : attributes.slice(*template_tokens)
+
+        if parameters.is_a?(Hash)
+          parameters.each do |key, value|
+            attributes["sentry.message.parameters.#{key}"] = value
+          end
+        else
+          parameters.each_with_index do |param, index|
+            attributes["sentry.message.parameters.#{index}"] = param
+          end
+        end
+      end
+    end
+
+    TOKEN_REGEXP = /%\{(\w+)\}/
```

Comment:
we should have a clear note in docs that our templates are expected to be like

```ruby
"This is a template with a %{var}"
```

and not what ruby people are used to 
```ruby
"This is a template with a #{var}"
```